src/components/Cart/CartInfo.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from '@emotion/styled';
+import { infoIcon } from '../../assets';
+
+interface CartInfoProps {
+  description?: string;
+  style?: React.CSSProperties;
+}
+
+function CartInfo({ description, style }: CartInfoProps) {
+  return (
+    <Container style={style}>
```

Comment:
이모션을 사용하고 있으니까, style대신에 css로 표현하면 좋을 것 같아요.

```suggestion
    <Container css={style}>
```

style대신에 css를 사용하는게 어떤 차이가 있는걸까요!?

Reply:
찾아보니 주요 차이점은 css를 사용할 경우에 동적 스타일링이 가능하고 의사 클래스, 미디어 쿼리 등의 요소를 사용할 수 있다는 점이 있네요! 
`css`를 사용하는 것으로 개선했어요!

다음은 AI에게 물어본 내용입니다.

### style prop vs css prop

1. style prop (인라인 스타일)

`<div style={{ marginTop: '32px' }}>`
  - React의 기본 인라인 스타일
  - JavaScript 객체로 스타일 정의
  - camelCase 속성명 사용 (marginTop)
  - 단위 자동 추가 (숫자만 쓰면 px 자동 추가)
  - 의사 클래스/요소 사용 불가 (:hover, ::before 등)
  - 미디어 쿼리 사용 불가

2. css prop (Emotion CSS)

`<div css={css`margin-top: 32px;`}>`
  - Emotion의 강력한 스타일링 기능
  - CSS 문법 그대로 사용
  - kebab-case 속성명 사용 (margin-top)
  - 의사 클래스/요소 사용 가능
  - 미디어 쿼리 사용 가능
  - 동적 스타일링 가능
  - 더 나은 성능 (스타일 캐싱)


[refactor: CartInfo에 Emotion css prop 적용으로 스타일링 개선](https://github.com/woowacourse/react-shopping-cart/pull/405/commits/853dc09c1375cd4c69e020e664334db2b41e6d0c)